### PR TITLE
Update all deps and fix bug

### LINKED ISSUE #160: Can't disable a configuration for channel
Hi All,
I got error while I was trying to disable the stripe configuration for a specific channel
{
  "message": "[\n  {\n    \"code\": \"too_small\",\n    \"minimum\": 1,\n    \"type\": \"string\",\n    \"inclusive\": true,\n    \"exact\": false,\n    \"message\": \"String must contain at least 1 character(s)\",\n    \"path\": [\n      \"configurationId\"\n    ]\n  }\n]",
  "code": -32600,
  "data": {
    "code": "BAD_REQUEST",
    "httpStatus": 400,
    "path": "paymentAppConfigurationRouter.mapping.update",
    "stack": null
  }
}
 

image

### LINKED ISSUE #177: Blank app configuration screen, https://store-ar9v8dsz.saleor.cloud/sw.js returned 403
image

### PR DESCRIPTION
**Pull request by typeofweb**
Fixes #177
Fixes #160

### CODE REVIEW

**Comment by witoszekdev** [comment]
@mmiszy there is an issue with Node version in GH Actions pipeline, please could you look at that?

**Comment by mmiszy** [comment]
@witoszekdev yes, of course, it was supposed to be a draft, my bad

**Review by peelar — APPROVED** [review]

File: src/modules/stripe/stripe-api.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { createLogger, redactError } from "@/lib/logger";
 
 export const getStripeApiClient = (secretKey: string) => {
   const stripe = new Stripe(secretKey, {
-    apiVersion: "2022-11-15",
+    apiVersion: "2023-10-16",
```
Comment (witoszekdev) [inline src/modules/stripe/stripe-api.ts:16]: Can we update the apiVersion just like that? Shouldn't we also update webhooks?

File: src/modules/webhooks/stripe-webhook.ts [hunk 2]
```diff
@@ -96,6 +96,7 @@ async function processStripeEvent({
 
   if (transactionEventReportResult.errors.length > 0) {
     const message = transactionEventReportResult.errors.map((err) => err.message).join("\n");
+    console.dir(transactionEventReportResult.errors);
```
Comment (witoszekdev) [inline src/modules/webhooks/stripe-webhook.ts:99]: Leftover console.log

**Review by witoszekdev — CHANGES_REQUESTED** [review]

File: src/modules/stripe/stripe-api.ts (hunk 1 above)
Comment (mmiszy) [reply src/modules/stripe/stripe-api.ts:16]: What do you mean by updating the webhooks? Do you have a link to the Stripe docs describing that?

Comment (krzysztofwolski) [reply src/modules/stripe/stripe-api.ts:16]: https://stripe.com/docs/upgrades

Are changes in payment intent relevant for us?
what about the error handling changes mentioned in the changelog?

Comment (mmiszy) [reply src/modules/stripe/stripe-api.ts:16]: Yeah, I read that and I don't see anything relevant to us 👍

**Review by witoszekdev — APPROVED** [review]

**Review by witoszekdev — APPROVED** [review]

**Pull request merged**